extra.html

```diff
@@ -58,54 +58,45 @@
 
        <kubernetes-topology-icon kind="Pod">
          <svg class="kube-topology">
-           <use xlink:href="#vertex-Pod" x="15" y="15"></use>
+           <g transform="translate(15, 15)" class="Pod">
+             <circle r="15"></circle>
+             <text y="7">P</text>
```

Comment:
This is the icon in the legend that you're touching here. 

You have to create the icons manually in the render event handler and insert the <image> tag there, or whatever you want to be at each node.